recipe-client-addon/test/unit/test_PreferenceExperiments.js

```diff
@@ -477,3 +478,45 @@ add_task(withMockExperiments(withMockPreferences(async function testInit(experim
     "init set the value for a default pref experiment",
   );
 })));
+
+// init should register telemetry experiments
+add_task(withMockExperiments(async function testInit(experiments) {
+  experiments["experiment1"] = experimentFactory({name: "experiment1", branch: "branch1"});
+  experiments["experiment2"] = experimentFactory({name: "experiment2", branch: "branch2"});
+
+  let currentExperiments = TelemetryEnvironment.getActiveExperiments();
```

Comment:
Also, given @Osmose's comment, I guess we should have tests that we don't set things as active in telemetry which really aren't.